.gitlab-ci.yml

```diff
@@ -67,3 +67,15 @@ Test AD Integration:
   script:
     - cd integrations/ad_integration/
     - pytest tests/
+
```

Comment:
Overvej et lint job for `exporters/sql_export/lc_for_jobs_db.py`. Både `black` og `isort`.

Reply:
Overvej det samme for test-koden `exporters/sql_export/tests/*`.

Reply:
Overvej det samme for test-koden `integrations/os2sync/tests/*`.

Reply:
Vi venter lige med et lint-job indtil vi kan køre black i de installerede os-pythoner. Der er ingen grund til at spærre for udvikling på udviklingsserverne. Det er derfor vi har dem.

Reply:
Det ville jo nok heller ikke spærre for noget, blot kræve et skridt før merge, hvor der laves en oprydning.

Reply:
Nemlig. Når det er fixet, så det opfører sig ens er der ikke længere brug for ci :-)